start-regular.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd /home/dwemer/xtts-api-server/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)